app/src/main/java/com/example/ridemetricsdksample/MainActivity.java

```diff
@@ -32,11 +99,21 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onTripStarted() {
                 // TODO set text "driving"
+                stringFile.write("TRIP STARTED \n");
+                Log.d(TAG, "TRIP STARTED");
             }
 
             @Override
             public void onTripEnded() {
                 // TODO set text "not driving"
+                stringFile.write("TRIP ENDED \n");
+                Log.d(TAG, "TRIP ENDED");
+            }
+
+            @Override
+            public void onTripEvent(String json) {
```

Comment:
It is not good that events return in json format but if we already do it we definitly should provide to client the json's format and possible values. But it will be better to replace the json by Java class if we have time for it.